BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01822-01
+2.0.0-prerelease-01901-01
```

Comment:
ops, what I need from buildtoos is the Pinvoke whitelist update. It needs to be at least 2.0.0-prerelease-01825-02. On master it is already 2.0.0-prerelease-01902-02 but release/uwp6.0 is still on 2.0.0-prerelease-01822-01 so I can drop it to merge to master but will need to update if we want this in release/uwp6.0. However, we can do the whitelist locally, let me go with that first.